docs/java-run-your-first-app.md

```diff
@@ -0,0 +1,160 @@
+---
+id: java-run-your-first-app
+title: Run your first Temporal application with the Java SDK
+sidebar_label: Run your first app
+---
+
+Welcome to the evolution of application development!
+
+This tutorial is aimed at developers who are new to Temporal and have some basic knowledge of Java. We recommend setting aside about 20 minutes to complete it. By following this tutorial you will learn how to run a Temporal Workflow application using the [Temporal Java SDK](https://github.com/temporalio/java-sdk) and you will also witness the fundamentally resilient nature of a Workflow, getting just a taste of the value that Temporal can provide.
+
+In case you are still wondering what [Temporal](/docs/overview) is, that is OK, there is a lot that can be said about it. In this tutorial we will focus on two of its main user facing components which are the Temporal server and its Java SDK.
+
+![Temporal server and SDK diagram](/img/docs/temporal-server-and-sdk-icons.png)
+
+Together they are a solution to the complexities that arise from modern application development needs. You can think of Temporal as a sort of "cure all" for the pains developers experience when trying to make reliable applications. Temporal provides reliability primitives right out of the box, such as automatic retries, timeouts, databases to track application states, rollbacks due to process failures, and more.
+
+Let's run your first Temporal Workflow application and forever change the way you approach application development.
+
+## Project setup &nbsp;&nbsp; ![](/img/docs/repair-tools.png)
```

Comment:
Ya, I was trying to avoid having to custom resize the images. These are 64px. The next smaller size is 32px. So if we go down this road, then to use a heading icon (from flaticon), I will need to custom resize any heading image to something like 45 or 50 px.

Reply:
I can't use an image tag that references css, because then a broken image link shows up in the page TOC - which is lame.